src/layout/Header/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import { IoMdArrowDropdownCircle } from 'react-icons/io';
+
+import styles from './index.module.css';
+import githubPic from '../../assets/images/github.png';
+import bell from '../../assets/images/notification.png';
+
+const Header = () => {
+    return (
+        <nav className="navbar navbar-expand-lg bg-dark">
+            <div className="container-fluid">
+                <a href="/">
```

Comment:
Yeah, just again with a tags, they work but I think react likes Links better as i saw wee little yellow error line

Reply:
Yes, it makes sense for this one as it is internal Link, even though at the moment there's no need for internal link but it can be there for just a show. Now it doesn't refresh the page either which is perfect for React but not so cool for our 1 page where nothing is happening.